docs/skillmap/beginner-skillmap.md

```diff
@@ -1,13 +1,14 @@
-# Beginner Guide to Arcade Games
-* name: Beginner's Guide to Arcade Games 
-* description: Quickly create your own clicker game, greeting cards, and arcade collector game experience.
-* infoUrl: /educator-info/basic-map-info.md
+# Beginner Arcade
+* name: Beginner's Guide to Arcade Games
+* description: Learn to create arcade games of your own by completing these tutorials.
+* infoUrl: skillmap/educator-info/basic-map-info.md
```

Comment:
small: you'll need to remove the `.md` part to get the docs page to render nicely

Reply:
Thanks. I missed that.